.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
.idea files should be all ignored... the whole directory... these are android studio settings....